code/modules/food_and_drinks/pizzabox.dm

```diff
@@ -178,28 +182,26 @@
 				return
 			boxes += add
 			newbox.boxes.Cut()
-			to_chat(user, span_notice("You put [newbox] on top of [src]!"))
 			newbox.update_appearance()
 			update_appearance()
 			user.regenerate_icons()
-			if(boxes.len >= 5)
-				if(prob(10 * boxes.len))
-					to_chat(user, span_danger("You can't keep holding the stack!"))
+			if(length(boxes) >= 5)
+				if(prob(10 * length(boxes)))
+					balloon_alert(user, "oops!")
```

Comment:
This is a weird balloon alert, I think it would work better as either a message to chat still, or a balloon alert to all.

Reply:
changed it to a to viewers